commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliAwsSecretsManagertParameters.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline;
+
+import tech.pegasys.web3signer.core.config.AwsAuthenticationMode;
+import tech.pegasys.web3signer.core.config.AwsSecretsManagerParameters;
+
+import picocli.CommandLine.Option;
+
+public class PicoCliAwsSecretsManagertParameters implements AwsSecretsManagerParameters {
```

Comment:
Why are these command line options needed? The config for AWS for a single key is driven by config file. This looks like it is part of the bulk key loading?